### PR TITLE
A few additions to the summary view

### PR DESCRIPTION
**Pull request by snopoke**
- add data node calculations
- add buttons to show / hide conditions and calculations
- add condtion and calculate icons
- allow showing labels or question IDs
- add button to clear property search
- improve sidebar nav styling

image

### CODE REVIEW

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by dimagimon** [comment]
:+1: No lint errors found.

File: corehq/apps/app_manager/templates/app_manager/summary_new.html [hunk 1]
```diff
@@ -6,6 +6,15 @@
 {% load compress %}
 {% load djangular_tags %}
 
+{% block stylesheets %}
+{% compress css %}
```
Comment (biyeun) [inline corehq/apps/app_manager/templates/app_manager/summary_new.html]: the stylesheets block is already being compressed...

Comment (biyeun) [inline corehq/apps/app_manager/templates/app_manager/summary_new.html]: ...but perhaps we want to move to more explicit compression

**Comment by biyeun** [comment]
curious that the build passed with a compression block inside a compression block (`{% stylesheets %}`)

File: corehq/apps/app_manager/templates/app_manager/summary_new.html (hunk 1 above)
Comment (snopoke) [inline corehq/apps/app_manager/templates/app_manager/summary_new.html]: I don't think it is already being compressed

Comment (biyeun) [inline corehq/apps/app_manager/templates/app_manager/summary_new.html]: you are correct! I mistakenly thought this was inheritinge from hqwebapp/base.html

**Comment by biyeun** [comment]
ignore previous comment. looks good!

**Pull request merged**